rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- According to the project requirements, the `:date,:book, :person` attributes should have getter and setter methods, please fix this 👍